packages/p2p/src/stores/my-ads-store.js

```diff
@@ -579,7 +622,10 @@ export default class MyAdsStore extends BaseStore {
             max_transaction: localize('Max limit'),
             min_transaction: localize('Min limit'),
             offer_amount: localize('Amount'),
-            price_rate: localize('Fixed rate'),
+            payment_info: localize('Payment instructions'),
+            price_rate: this.root_store.floating_rate_store.rate_type
```

Comment:
`price_rate` isn't clear, please change it to `rate_type`.

Reply:
The key used in the form for identifying the floating-rate and fixed rate field. Will need to change in a lot of places. Please advice